build_defs.bzl

```diff
@@ -71,13 +73,20 @@ def nanobind_extension(
         NANOBIND_DOMAIN = ["NB_DOMAIN={}".format(domain)]
     else:
         NANOBIND_DOMAIN = []
+
+    linkstatic = kwargs.pop("linkstatic", True)
```

Comment:
this probably can't be the way to configure this as if the library has other deps it will affect them as well?